lambda_functions/transform_lambda.py

```diff
@@ -170,7 +170,10 @@ def get_resource_tags_from_metric(
                 arn = f"arn:aws-us-gov:rds:{region}:{account_id}:db:{db_name}"
                 # copy avoids mutating the cached value returned by get_tags_from_arn
                 result_tags = get_tags_from_arn(arn, rds_client).copy()
-                if metric.get("metric_name") == "FreeStorageSpace":
+                if (
+                    result_tags != {}
```

Comment:
```suggestion
                    result_tags
```

If `result_tags` is not an empty dict, `{}`, this will return `true`

Reply:
Tested it locally and works with test